api/clients/v2/cert_builder.go

```diff
@@ -113,6 +127,20 @@ func (cb *CertBuilder) getNonSignerStakesAndSignature(
 	rbn := signedBatch.GetHeader().GetReferenceBlockNumber()
 
 	// 3 - call operator state retriever to fetch signature indices
+	nonSignerOperatorIDsHex := make([]string, len(nonSignerOperatorIDs))
+	for i, id := range nonSignerOperatorIDs {
+		nonSignerOperatorIDsHex[i] = "0x" + hex.EncodeToString(id[:])
+	}
+	// We log the call parameters for debugging purposes. If execution reverts, we can debug using tenderly.
+	cb.logger.Info("eth-call",
```

Comment:
How would debugging work? Do I have to manually convert to calldata or something like that?